.circleci/images/primary/Dockerfile-2.1.10

```diff
@@ -4,7 +4,7 @@
 # Note: See the "Publishing updates to images" note in ./README.md for how to publish new builds of this container image
 
 # taken from https://github.com/docker-library/ruby/blob/c5693b25aa865489fee130e572a3f11bccebd21b/2.1/Dockerfile
-FROM buildpack-deps:stretch AS ruby-2.1.10-stretch
+FROM buildpack-deps:bookworm AS ruby-2.1.10-bookworm
```

Comment:
If anyone else is wondering [stretch is debian 9, and bookworm is debian 12](https://distrowatch.com/table.php?distribution=debian) so this buys us a lot of time! Nice :D